### DISCUSSION TITLE
No permission, please contact the administrator

### DISCUSSION THREAD

**Discussion by Tommyvane** [apache/streampark#1226] [Q&A]
i has been deployed this application， login by admin / streamx ；but it response this title ：No permission, please contact the administrator, i am looking for reply.

![image](https://user-images.githubusercontent.com/44920155/178993910-10e01644-f424-4249-a919-8935d280c47e.png)

**Comment by hujunhui530** [comment]
不要执行upgrade.sql  ，只执行 final.sql
do not execute upgrade.sql ，execute final.sql
